combinations.py

```diff
@@ -0,0 +1,30 @@
+# itertools.combinations()
+"""Task
+You are given a string S.
+Your task is to print all possible combinations, up to size k, of the string in lexicographic sorted order.
+
+Input Format
+A single line containing the space separated string S and the integer value k.
+
+Constraints
+* 0 < k <= len(S)
+* The string contains only UPPERCASE characters.
+
+Output Format
+Print the different combinations of string S on separate lines.
+"""
+
+
+from itertools import combinations
+
+if __name__ == '__main__':
+    s, k = input('please, enter the string and the integer value: ').split()
+    try:
+        if 0 < int(k) <= len(s):
+            for i in range(1, int(k) + 1):
+                for l in list(combinations(sorted(s.upper()), i)):
```

Comment:
1) `upper` is not necessary - the input constraints say it's already uppercase
2) `list` is not necessary - for loop will yield a tuple containing combination, one by time
3) `combinations` retain lexicographic order, I'm not sure if the task implies sorting, please run tests without sorted and see if it passes

Reply:
'Sorted' is necessary because the tests don't pass without it (I've checked out); the rest is fixed